components/core/blocks/Checkout/ThankYouPage.vue

```diff
@@ -126,7 +126,7 @@
 
 <script>
 import Composite from '@vue-storefront/core/mixins/composite';
-import Breadcrumbs from 'theme/components/core/Breadcrumbs';
+import Breadcrumbs from './Breadcrumbs';
```

Comment:
I moved Breadcrumbs component to Checkout folder, because it's used only in ThankYouPage and it will be probably removed in future (we want to use SfBreadcrumbs)

Reply:
ok